lib/device_api/android/device.rb

```diff
@@ -253,6 +253,13 @@ def diskstat
       def uptime
         ADB.get_uptime(serial)
       end
+
+      # Returns the Wifi mac address
+      def wifi_mac_address
+        network = ADB.get_network_info(serial)
+        wifi = network.select { |a| a[:name] ==  'wlan0' }.first
```

Comment:
Use `find` instead of `select.first`.<br>Unnecessary spacing detected.<br>Operator `==` should be surrounded with a single space.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.